composer.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "easir/codestyle-php",
-    "require-dev": {
+    "require": {
+        "php": ">=7",
```

Comment:
We _could_ do that, but why would we?
5.6 is EOL http://php.net/supported-versions.php